persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_54_gsheet/generated/entities/persist_client.bal

```diff
@@ -1,11 +1,9 @@
+import ballerina/http;
 // AUTO-GENERATED FILE. DO NOT MODIFY.
-
 // This file is an auto-generated file by Ballerina persistence layer for model.
 // It should not be modified by hand.
```

Comment:
Shall we move comments ahead of imports

Reply:
Updated in [a2dce9b](https://github.com/ballerina-platform/persist-tools/pull/303/commits/a2dce9b94c412beece63be75d51ec8676f20f1a5)